extensions/lark/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "lark",
+  "title": "Lark",
```

Comment:
Since Lark is a big suite of apps, give your extension more specific title. E.g. "Lark Documents"

Please check out [these guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store), it should help.